Automattic/wp-calypso
# Jetpack Manage: Add feature flag and menu item for URL connection

Related to https://github.com/orgs/Automattic/projects/876/views/1?pane=issue&itemId=50406555
Related to https://github.com/orgs/Automattic/projects/876/views/1?pane=issue&itemId=50406661

## Proposed Changes

* Adds a feature flag called `jetpack/url-only-connection` for gating URL connection features, and enables that flag only in development environments
* Adds a menu item in the "Add site" dropdown on the Jetpack Manage dashboard
* Also - **nitpick** - fixes order of feature flags as `jetpack/card-addition-improvements` was not alphabetized properly

## Testing Instructions

* Ensure that the "Add sites by URL" menu item appears in a dev environment, but not in Jetpack Cloud live environment

## Pre-merge Checklist
